mmv1/products/developerconnect/GitRepositoryLink.yaml

```diff
@@ -12,124 +12,138 @@
 # limitations under the License.
 
 ---
-name: 'GitRepositoryLink'
-description: "A git repository link to a parent connection."
-min_version: 'beta'
-docs:
-id_format: 'projects/{{project}}/locations/{{location}}/connections/{{parent_connection}}/gitRepositoryLinks/{{git_repository_link_id}}'
-base_url: 'projects/{{project}}/locations/{{location}}/connections/{{parent_connection}}/gitRepositoryLinks'
-self_link: 'projects/{{project}}/locations/{{location}}/connections/{{parent_connection}}/gitRepositoryLinks/{{git_repository_link_id}}'
-create_url: 'projects/{{project}}/locations/{{location}}/connections/{{parent_connection}}/gitRepositoryLinks?gitRepositoryLinkId={{git_repository_link_id}}'
+name: GitRepositoryLink
```

Comment:
Do you mind moving the changes to this file to a separate PR? This PR is too large and difficult to review. It is easy to missing something during the code review.

Reply:
Removed all changes to and associated test files. Kept change to promote to GA so as to not break the provider.